### DISCUSSION TITLE
Workaround for bind volume issue on Mac with qemu/podman patch

### DISCUSSION THREAD

**Discussion by jeesmon** [containers/podman#13257] [General]
Creating this thread to discuss/help/support people using qemu/podman patch from @willcohen and @philnalwalker posted in https://github.com/containers/podman/issues/8016 so we don't mask original issue with support requests for the workaround.

* Instructions to use patched version of qemu/podman to get bind volume working with podman machine on Mac: https://github.com/containers/podman/issues/8016#issuecomment-1034091359
* If you don't want to build qemu from source, you can use this tap: https://github.com/containers/podman/issues/8016#issuecomment-1047905064
* Some updates to the instructions from @lehors : https://github.com/containers/podman/issues/8016#issuecomment-1034918559
* Workaround for permission issues suggested by @afbjorklund : https://github.com/containers/podman/issues/8016#issuecomment-1037068684
* Steps to run kind/minikube with patched podman : https://github.com/containers/podman/issues/8016#issuecomment-1040522824 and https://github.com/kubernetes/minikube/issues/13618#issuecomment-1040759499

**Comment by jeesmon** [comment]
@jmb-missionlane Let's discuss here if you run into more issues with the patch

**Comment by grafanalf** [comment]
I needed this to get `qemu-9fps` to build:

```
diff --git a/Formula/qemu.rb b/Formula/qemu.rb
index 37952e1da28..dc1f2365be9 100644
--- a/Formula/qemu.rb
+++ b/Formula/qemu.rb
@@ -42,7 +42,7 @@ class Qemu < Formula

   # 820KB floppy disk image file of FreeDOS 1.2, used to test QEMU
   resource "homebrew-test-image" do
-    url "https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/distributions/1.2/FD12FLOPPY.zip"
+    url "https://www.ibiblio.org/pub/micro/pc-stuff/freedos/files/distributions/1.2/official/FD12FLOPPY.zip"
     sha256 "81237c7b42dc0ffc8b32a2f5734e3480a3f9a470c50c14a9c4576a2561a35807"
   end
```
